dsgrid/registry/project_registry_manager.py

```diff
@@ -232,6 +239,37 @@ def _submit_dataset(
             project_config.config_id,
         )
 
+    def check_dataset_base_to_base_mappings(self, project_config, dataset_config, dataset):
```

Comment:
Should this be check_dataset_base_to_project_base_mappings?

Reply:
Yes